Error with SIGXCPU (mingw)

I have this error in mingw (for 1.4.0 patch):

```
C:/msys64/usr/local/precice-1.4.0/src/precice/impl/SolverInterfaceImpl.cpp:80:10: error: 'SIGXCPU' was not declared in this scope signal(SIGXCPU, precice::utils::terminationSignalHandler);
```

I solved this by adding (in SolverinterfaceImpl.cpp):
```
#define	SIGXCPU	24	/* exceeded CPU time limit */
```

[SolverInterfaceImpl.zip](https://github.com/precice/precice/files/2940846/SolverInterfaceImpl.zip)

--- a/src/precice/impl/SolverInterfaceImpl.cpp
+++ b/src/precice/impl/SolverInterfaceImpl.cpp
@@ -32,6 +32,10 @@
 #include "versions.hpp"
 
 #include <csignal> // used for installing crash handler
+#ifndef SIGXCPU
+#define SIGXCPU 24 /* exceeded CPU time limit */
+#endif
+
 #include <utility>
 
 #include "logging/Logger.hpp"

